### ISSUE TITLE
Create User SignUp screen

### ISSUE THREAD

**Issue by ReallyBoard123** [devhausleipzigacademy/camp10-midterm#73]
We use this screen to collect information from the user. 
It hosts a form that is connected to our database and once the information provided matches our criteria. The user is created. 

If successful, it navigates the user to `/login`

If not, it displays the error using `https://react-hot-toast.com/` ^

Design is a combination of two separate pages (somehow mixed)

First part:

- [ ] Upload userImage component (check this: https://www.figma.com/file/ofN4fC7BH5gbJbhwMoCZGS/Untitled?type=design&node-id=25%3A431&mode=design&t=kEJ1rbxZTrFeXZRo-1)

Warning: Currently we are not able to store this information anywhere. In a separate issue, we will use a database to be able to store the user-uploaded images and store the link to our database. For now, you could also leave it empty with a placeholder.

Second part (https://www.figma.com/file/ofN4fC7BH5gbJbhwMoCZGS/Untitled?type=design&node-id=26%3A561&mode=design&t=kEJ1rbxZTrFeXZRo-1) <- look at the form part of it. Ignore the user profile picture and the three tabs.

1. Form component - use the input Component that we already have.

make the following inputs
- [ ] Firstname
- [ ] Lastname
- [ ] Email
- [ ] City
- [ ] Plz

2. Submit Button

Once this front-end part is complete, you could also quickly finish #60 and submit both tasks together!!
